content/tutorial/tutorial.md

```diff
@@ -287,17 +292,17 @@ class Square extends React.Component {
 }
 ```
 
->Note
+>ध्यान दें
 >
->In [JavaScript classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes), you need to always call `super` when defining the constructor of a subclass. All React component classes that have a `constructor` should start it with a `super(props)` call.
+>[जावास्क्रिप्ट क्लासेस](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes) में, उप-वर्ग के निर्माणकर्ता को परिभाषित करते समय आपको हमेशा `super` कॉल करने की आवश्यकता होती है। सभी React कौम्पोनॅन्ट क्लासेस जिनके पास `कंस्ट्रक्टर` है, उन्हें `super(props)` कॉल से शुरू करना चाहिए।
```

Comment:
उप-वर्ग would be better as subclass?

Reply:
निर्माणकर्ता  -> constructor

Reply:
> उप-वर्ग would be better as subclass?

I can't think of any other better option for this

Reply:
the word subclass in hindi?